### PR TITLE
updated timestamp format to ISO string in realtime payload

### PR DESCRIPTION
**Pull request by Akshay-Rana-Gujjar**
## What does this PR do?

Fixed: Invalid Timestamp in Realtime Payload
The timestamp is an ISO string like 2022-10-10T13:37:13.780+00:00

## Test Plan

Created a project using web sdk & created test database in console to check if timestamp value returing from reatime payload is in ISO format.

## Related PRs and Issues

https://github.com/appwrite/appwrite/issues/4446

### Have you added your change to the [Changelog](https://github.com/appwrite/appwrite/blob/master/CHANGES.md)?

No

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Yes

### CODE REVIEW

**Comment by Akshay-Rana-Gujjar** [comment]
`{
    "events": [
        "databases.634c7d4718c18a008124.collections.634c7d4a8460c867f767.documents.A.update",
        "databases.*.collections.*.documents.*.update",
        "databases.634c7d4718c18a008124.collections.*.documents.A.update",
        "databases.*.collections.*.documents.A.update",
        "databases.*.collections.634c7d4a8460c867f767.documents.A.update",
        "databases.634c7d4718c18a008124.collections.634c7d4a8460c867f767.documents.*.update",
        "databases.*.collections.634c7d4a8460c867f767.documents.*.update",
        "databases.634c7d4718c18a008124.collections.*.documents.*.update",
        "databases.634c7d4718c18a008124.collections.634c7d4a8460c867f767.documents.A",
        "databases.*.collections.*.documents.*",
        "databases.634c7d4718c18a008124.collections.*.documents.A",
        "databases.*.collections.*.documents.A",
        "databases.*.collections.634c7d4a8460c867f767.documents.A",
        "databases.634c7d4718c18a008124.collections.634c7d4a8460c867f767.documents.*",
        "databases.*.collections.634c7d4a8460c867f767.documents.*",
        "databases.634c7d4718c18a008124.collections.*.documents.*",
        "databases.634c7d4718c18a008124.collections.634c7d4a8460c867f767",
        "databases.*.collections.*",
        "databases.634c7d4718c18a008124.collections.*",
        "databases.*.collections.634c7d4a8460c867f767",
        "databases.634c7d4718c18a008124",
        "databases.*"
    ],
    "channels": [
        "documents",
        "databases.634c7d4718c18a008124.collections.634c7d4a8460c867f767.documents",
        "databases.634c7d4718c18a008124.collections.634c7d4a8460c867f767.documents.A"
    ],
    "timestamp": "2022-10-17T07:22:48+00:00",
    "payload": {
        "name": "1665991367789",
        "$id": "A",
        "$createdAt": "2022-10-16T21:53:51.678+00:00",
        "$updatedAt": "2022-10-17T07:22:47.956+00:00",
        "$permissions": [],
        "$collectionId": "634c7d4a8460c867f767",
        "$databaseId": "634c7d4718c18a008124"
    }
}`

This is the response I am getting, please check timestamp value.

Please let me know if any change required. Thanks

cc: @stnguyen90

File: src/Appwrite/Messaging/Adapter/Realtime.php [hunk 1]
```diff
@@ -149,7 +149,7 @@ public static function send(string $projectId, array $payload, array $events, ar
             'data' => [
                 'events' => $events,
                 'channels' => $channels,
-                'timestamp' => DateTime::now(),
+                'timestamp' => date('c'),
```
Comment (stnguyen90) [inline src/Appwrite/Messaging/Adapter/Realtime.php:152]: I know it's longer than what you have, but let's do the following to be consistent with what we do in the rest of the codebase:

```suggestion
                'timestamp' => DateTime::formatTz(DateTime::now()),
```

**Review by stnguyen90 — CHANGES_REQUESTED** [review]
Would you please also:

1. include a screenshot of the new timestamp format
2. Update the changelog as the PR template asks

**Comment by Akshay-Rana-Gujjar** [comment]
@stnguyen90  I have updated the code with your suggested changes.

Also I have attached the screenshot of timestamp ISO format.


Screenshot from 2022-10-18 22-40-03

**Comment by Akshay-Rana-Gujjar** [comment]
@stnguyen90 a you suggest changes about chnagelog, what should I add there?

**Comment by Akshay-Rana-Gujjar** [comment]
@stnguyen90 could you please help me here?

**Review by stnguyen90 — CHANGES_REQUESTED** [review]
> @stnguyen90 could you suggest changes about chnagelog, what should I add there?

@Akshay-Rana-Gujjar, the CHANGES.md file tracks all the changes that make it to the `main` branch. Add your change to [this](https://github.com/appwrite/appwrite/blob/master/CHANGES.md) file in the following format:

```
- One line description of your PR [#pr_number](Link to your PR)
```

Would you please also change the base of your PR to the 1.0.x branch?

**Comment by PineappleIOnic** [comment]
Thank you so much for the PR 🤩. We're adding the `hacktoberfest-accepted` label to ensure this PR counts towards your Hacktoberfest contributions count. With that said, please stay active on this PR to address any comments once you receive a review. Happy Hacktoberfest! 🎃

**Comment by Akshay-Rana-Gujjar** [comment]
Hi @stnguyen90, after changing the base there are lot of changes and 1 conflict in CHANGES.md. Do I need to create a new branch from new base branch that is 1.0.x and made my changes there and override all the changes in my current branch ([Akshay-Rana-Gujjar:fix-4446-updated-timestamp-format-to-ISO-in-realtime-payload](https://github.com/Akshay-Rana-Gujjar/appwrite/tree/fix-4446-updated-timestamp-format-to-ISO-in-realtime-payload))? or could you suggest other way to do this?

**Comment by stnguyen90** [comment]
@Akshay-Rana-Gujjar, you don't need to change the base branch. And no, you wouldn't create another branch to fix merge conflicts.

**Comment by stnguyen90** [comment]
@Akshay-Rana-Gujjar any update on this?

**Comment by Akshay-Rana-Gujjar** [comment]
> @Akshay-Rana-Gujjar any update on this?

Hi @stnguyen90, I can start working on this task again. I was busy with my office work. Now I can spare some time for it.

**Review by stnguyen90 — APPROVED** [review]

**Review by abnegate — APPROVED** [review]

**Comment by Akshay-Rana-Gujjar** [comment]
Hi all, sorry for not active for a while. Recently I have joined a new company I am settling down there.

**Comment by stnguyen90** [comment]
@Akshay-Rana-Gujjar, actually, I think this is all good from your end. We just need a final review and then merge.

**Review by christyjacob4 — APPROVED** [review]

**Pull request merged**